app/Http/Controllers/EmployeeController.php

```diff
@@ -53,7 +53,17 @@ public function store(Request $request){
     }
 
     public function update(Request $request , string $id){
-        return('admin');
+
+        Employee::find($id)->update([
+            'name'=>$request->name,
+            'position'=>$request->position,
+            'dob'=>$request->dob,
```

Comment:
I saw that you have made the migration with `date` type for column `dob` what will happened if the request send with other values except date format

Reply:
![image](https://github.com/user-attachments/assets/9fb4d612-2edb-4680-b9c8-2a0ecefd6ff7)

throws error.....form not submitted #22 @Raza9798 

shall i add backend validations?

Reply:
yes please add it @axnish27

Reply:
okay sure @Raza9798

Reply:
Validation for both edit and create functions implemented please review @Raza9798 #22